backends/platform/sdl/win32/win32.h

```diff
@@ -38,6 +38,8 @@ class OSystem_Win32 : public OSystem_SDL {
 
 	virtual bool openUrl(const Common::String &url);
 
+	static Common::String getScreenshotsPath();
```

Comment:
Maybe this could be made a virtual function in OSystem_SDL so that if needed we can specify a location on other OS's as well.